docs/how-to-run.adoc

```diff
@@ -60,6 +60,34 @@ your hardware. Much of this configuration will be automated in the future
 automatically be detected on discovered disks), but for now it remains
 hard-coded.
 
+=== Getting into your guests
+
+Omicron currently implements a bit of a hack to allow external connectivity into
+guest instances. This overrides the default behavior of the Oxide Packet
+Transformation Engine (OPTE, the kernel module which provides the private
+virtual networking to guests). To make this work, OPTE needs to know about the
+local networking configuration, in this case the MAC address of the local
+internet gateway. To be able to get into your instances, you _must_ specify this
+in the `gateway_mac` field of the config file `smf/sled-agent/config-rss.toml`.
+
+The value there is correct for the lab environment, so if you're running there,
+no changes are needed. If you're running elsewhere, you find the value with:
+
+[source,text]
+----
+$ netstat -rn | grep default
+default              192.168.1.1          UG       10     431926 igb0
+$ arp -an | grep -w 192.168.1.1
+igb0   192.168.1.1          255.255.255.255          74:ac:b9:a4:dc:02
+----
```

Comment:
That's because folgers is a bit special in that it's designed not to have internet access. So in that case you must have knowledge of the network it's on. For anyone else on their home network they will have a default route.